lib/temporal/testing/scheduled_workflows.rb

```diff
@@ -0,0 +1,75 @@
+module Temporal
+  module Testing
+    class WorkflowIDNotScheduled < ClientError; end
+
+    # When Temporal.schedule_workflow is called in a test in local mode, we defer the execution and do
+    # not do it automatically.
+    # You can execute them or inspect their cron schedules using this module.
+    module ScheduledWorkflows
+      def self.execute(workflow_id:)
+        Private::Store.execute(workflow_id: workflow_id)
+      end
+
+      def self.execute_all
+        Private::Store.execute_all
+      end
+
+      # For someone who wants to assert that the schedule is what they expect.
+      # Populated by Temporal.schedule_workflow
+      # format: { <workflow_id>: <cron schedule string>, ... }
+      def self.cron_schedules
+        Private::Store.schedules
+      end
+
+      def self.clear_all
+        Private::Store.clear_all
+      end
+
+      module Private
```

Comment:
You can make a class/module private with `private_constant`, but that doesn't work using it from two modules.  This is a convention we use in our codebase (though we have tooling to enforce it.)
Open to better options.